packages/faustwp-cli/src/index.ts

```diff
@@ -102,33 +67,12 @@ import {
     }
   }
 
-  if (shouldFireTelemetryEvent()) {
-    try {
-      const wpTelemetryData = await requestWPTelemetryData(
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        getWpUrl()!,
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        getWpSecret()!,
-      );
-
-      if (!wpTelemetryData) {
-        throw new Error(
-          'There was a problem retrieving telemetry data from the WordPress instance',
-        );
-      }
-
-      const telemetryData = marshallTelemetryData(wpTelemetryData, arg1);
-
-      debugLog('Telemetry event: ', telemetryData);
-
-      void sendTelemetryData(
-        telemetryData,
-        userConfig.get('telemetry.anonymousId') as string,
-      );
-    } catch (err) {
-      // console.log(err);
-      // Fail silently
-    }
+  try {
+    const telemetryData = marshallTelemetryData(arg1);
+    void sendTelemetryData(telemetryData);
+  } catch (err) {
```

Comment:
You may need to check here if the secret key exists or not. We do not want to fire off a telemetry event without the secret key in the header.